### PR TITLE
Added recipe for extras

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @rbtcollins in case he would like to be added as a maintainer to the `extras` builder on conda-forge, a library of community-build `conda` packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/extras-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that is entirely fine too.

### CODE REVIEW

File: recipes/extras/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{%set name = "extras" %}
+{%set version = "1.0.0" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "" %}
```
Comment (ocefpaf) [inline recipes/extras/meta.yaml]: :confounded:

**Comment by ocefpaf** [comment]
I believe you missed the hash value :smile:

File: recipes/extras/meta.yaml (hunk 1 above)
Comment (pmlandwehr) [inline recipes/extras/meta.yaml]: Jinjaaaa!

Comment (ocefpaf) [inline recipes/extras/meta.yaml]: Hehe. We actually need to fix the linter to catch that. It is a hard to review!

**Pull request merged**